features/support/hapybara.rb

```diff
@@ -22,7 +22,7 @@
   # https://github.com/teamcapybara/capybara/issues/1890
   # Google Chrome 71.0.3578.80
   require 'webdrivers'
-  Webdrivers::Chromedriver.version = '2.46'
+  # Webdrivers::Chromedriver.version = '2.46'
```

Comment:
Was this commented out by mistake? Some tests are failing in CI. I couldn't quickly find out why except if it is because of this change.

I remember there were some issues with running tests locally with Chrome. If this was commented out because of that, it might be that the driver change requires some changes to the failing tests, so they would pass in CI.

Reply:
Yes, I forgot to uncomment, locally my tests did not run on this version of the driver.